src/components/collection-card/components/with-auth-elements/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { useMemo } from 'react'
+import { useCollectionProgress } from 'hooks/progress'
+import {
+	CollectionProgressStatusElements,
+	parseCollectionProgress,
+} from 'components/collection-progress-group'
+import CollectionCard, {
+	CollectionCardPropsWithId,
+} from 'components/collection-card'
+import s from './with-auth-elements.module.css'
+
+/**
+ * Displays a collection card, with progress elements where possible.
+ *
+ * Works without authentication enabled, displaying a tutorial count
+ * similar to the tutorial count displayed on plain collection cards.
+ *
+ * With authentication enabled, for authenticated users with progress,
+ * we display a progress bar or "complete" at the top of the card.
+ */
+function CollectionCardWithAuthElements({
```

Comment:
`CollectionCardWithAuthElements` wraps `CollectionCard`, using the newly introduced `eyebrowSlot` to display user progress elements (Very similar to `TutorialCard`).